src/Bicep.LangServer/Completions/BicepCompletionProvider.cs

```diff
@@ -217,21 +217,22 @@ private IEnumerable<CompletionItem> GetModulePathCompletions(SemanticModel model
             // "./" will not be preserved when making relative Uris. We have to go and manually add it.
             // Prioritize .bicep files higher than other files.
             var fileItems = files
-                .Where(file => file.Segments.Last().EndsWith(LanguageServerConstants.LanguageId))
+                .Where(file => file != model.SyntaxTree.FileUri)
+                .Where(file => file.Segments.Last().EndsWith("." + LanguageServerConstants.LanguageId))
```

Comment:
Can you create a new constant for the file extension with the value ".bicep"? LanguageId doesn't necessarily have to match the extension (especially if we end up renaming the language for some reason).